docs/modules/mssql.md

```diff
@@ -0,0 +1,63 @@
+# MS SQL Server
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for MSSQLServer.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the MSSQLServer module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/mssql
+```
+
+## Usage example
+
+<!--codeinclude-->
```

Comment:
Not sure why this code snippet looks "double spaced" when it renders on the deployed webpage.


[See here](https://deploy-preview-1969--testcontainers-go.netlify.app/modules/mssql/)